include/ImmediateContext.hpp

```diff
@@ -918,6 +918,7 @@ class ImmediateContext
         GUID CreatorID;
         DWORD MaxAllocatedUploadHeapSpacePerCommandList;
         DWORD MaxSRVHeapSize;
+        DWORD BufferPoolTrimThreshold;

```

Comment:
Should we set a default value for this in the constructor so callers don't all have to change? Otherwise callers that aren't updated perfectly in sync will start getting 5 (clamped from 0) instead of 100.

Reply:
Yep, absolutely right. Brain was focused on 9on12 defaulting to DWORDMAX